docs/how-to-guides/index.md

```diff
@@ -8,6 +8,9 @@ Simple and short articles grouped by topics, each introduces a core feature of p
 tracing/index
 ```
 
+```{toctree}
```

Comment:
What is this trick?

Reply:
docsite magic, support views like:

https://microsoft.github.io/promptflow/how-to-guides/develop-a-dag-flow/index.html

![image](https://github.com/microsoft/promptflow/assets/38847871/e58c329f-d732-48b9-861c-7b7dbabe06ae)

Reply:
why empty toctree?

Reply:
by mistake, deleted.